### PR TITLE
Add documentation about DefaultStorageClass plugin.

### PR DESCRIPTION
**Pull request by jsafrane**
See pr #1064 for related volume documentation updates.

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1087)

### CODE REVIEW

**Comment by jaredbhatti** [comment]
Can you rebase?

**Comment by jsafrane** [comment]
rebased

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -140,10 +140,31 @@ and ensures that requests in a non-existent `Namespace` are rejected.
 A `Namespace` deletion kicks off a sequence of operations that remove all objects (pods, services, etc.) in that
 namespace.  In order to enforce integrity of that process, we strongly recommend running this plug-in.
 
+### DefaultStorageClass
+
+This plug-in observes creation of `PersistentVolumeClaim` objects that do not require any specific storage class
```
Comment (thockin) [inline docs/admin/admission-controllers.md]: s/require/request

File: docs/admin/admission-controllers.md [hunk 2]
```diff
@@ -140,10 +140,31 @@ and ensures that requests in a non-existent `Namespace` are rejected.
...
+and automatically adds a default storage class to them.
+This way, users that do not request any special storage class do no need to care about them at all and they
+will get the default one.
+
+This plug-in does not do anything when no default storage class is configured. When more than one storage
```
Comment (thockin) [inline docs/admin/admission-controllers.md]: cover how to mark an SC as default?

Comment (jsafrane) [inline docs/admin/admission-controllers.md]: just few lines below there is "See [persistent volume] documentation about [...] how to mark a storage class as default."

I'd like to have it on one place, as it is probably going to evolve.

File: docs/admin/admission-controllers.md (hunk 1 above)
Comment (jsafrane) [inline docs/admin/admission-controllers.md]: fixed

**Comment by jsafrane** [comment]
rebased

**Comment by devin-donnelly** [comment]
Is this change relevant now, or will it only be relevant after the 1.4 release?

**Comment by saad-ali** [comment]
StorageClass are added as beta API objects starting in v1.4. So both this PR and #1064 will only be relevant after the v1.4 release.

**Comment by devin-donnelly** [comment]
Okay. We're trying to wrangle all 1.4-related docs changes in the release branch (called "release-1.4"). Would you mind changing the target branch for this PR to release-1.4 ?

**Comment by saad-ali** [comment]
> Okay. We're trying to wrangle all 1.4-related docs changes in the release branch (called "release-1.4"). Would you mind changing the target branch for this PR to release-1.4 ?

@jsafrane Could you do this please.

@devin-donnelly Looks like PR https://github.com/kubernetes/kubernetes.github.io/pull/1064 went in to the master branch. Is that ok? Should it be cherry-picked to the release-1.4 branch?

**Comment by jsafrane** [comment]
rebased on top of branch release-1.4

**Comment by janetkuo** [comment]
This PR targets master so shouldn't rebase on release-1.4. You may want to open another PR targeting 1.4 branch instead.

**Comment by jsafrane** [comment]
Ok, I'm rebasing this PR back to master and I'll open a new one for 1.4 branch. Feel free to close any of them.

**Comment by devin-donnelly** [comment]
Closing as duplicate of #1218.

**Pull request closed without merging**